homeassistant/components/aladdin_connect/cover.py

```diff
@@ -75,6 +75,7 @@ def __init__(self, acc: AladdinConnectClient, device: DoorDevice) -> None:
         self._number = device["door_number"]
         self._attr_name = device["name"]
         self._attr_unique_id = f"{self._device_id}-{self._number}"
+        self._attr_is_closed = bool(STATES_MAP.get(device["status"]))
```

Comment:
Why is this differently implemented compared to the `update()` method?

Reply:
I just saw this: https://github.com/home-assistant/core/pull/54015 It appears to fix this already.

Reply:
Yup, indeed, let's close this one. 👍